src/Random.cpp

```diff
@@ -93,20 +85,13 @@ int RNG::RandInt(int aX, int aY)
             return aY;
         }
     }
-    return aX + (rand() % (aY - aX + 1));
-#endif
-    
+    return aX + (rand() % (aY - aX + 1)); 
```

Comment:
using `rand()` is actually an issue for reproducibility

Reply:
Sure, but I think we need to revise the whole random number generation. I'd rather pass a callback that is requests a random number, so I can keep track of it myself. The original boost implementation was also very inconvient wrt reproducability.

Reply:
should be resolved now